packages/rest/src/rest-http-error.ts

```diff
@@ -12,7 +14,15 @@ export namespace RestHttpErrors {
     const msg = `Parameters with "in: ${location}" are not supported yet.`;
     return new HttpErrors.NotImplemented(msg);
   }
-  export function invalidRequestBody(msg: string): HttpErrors.HttpError {
-    return new HttpErrors.UnprocessableEntity(msg);
+  export function invalidRequestBody(): HttpErrors.HttpError {
+    return new HttpErrors.UnprocessableEntity(
+      'The request body is invalid. See error object `details` property for more info.',
```

Comment:
Shall we extract out the message into a variable like the other functions above?

Reply:
👍 definitely.